src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -207,6 +207,27 @@ entity corresponds to modality suffix,
 such as `T1w` or `inplaneT1`, referenced by the defacemask image.
 For example, `sub-01_mod-T1w_defacemask.nii.gz`.
 
+The OPTIONAL [`task-<label>`](../99-appendices/09-entities.md#task) entity can be used
```

Comment:
```suggestion
The OPTIONAL [`task-<label>`](../99-appendices/entities.md#task) entity can be used
```

We have removed the number prefix for those files:

https://github.com/bids-standard/bids-specification/blob/master/src/appendices/entities.md

Reply:
Ah this is probably a leftover from my fork of the spec not being up to date...let me also update it.